tools/configure

```diff
@@ -38,6 +38,7 @@ app_opts() ->
     %% Opt         Apps                 Desc
     [{"none",      [],                  "include no 3rd party drivers"},
      {"all",       [],                  "include all drivers"},
+     {"fast_tls",  ["fast_tls"],        "include tls library"},
```

Comment:
There is no need for this option in the `configure` script as we need this lib in every build.